src/scss/_nav.scss

```diff
@@ -16,6 +16,7 @@
 
 	@include oGridRespondTo(M) {
 		position: relative;
+		left: 20px;
```

Comment:
Would it be worth making this a variable? As it's the button size, which would mean it could be updated in multiple places.